client/src/component/Login/SecondFactorForm.tsx

```diff
@@ -4,44 +4,32 @@ import login2F from 'corla/action/login2F';
 
 
 function isFormValid(form: Form): boolean {
-    const token = form.tokenParts.join('');
+    return form.token !== '';
+}
 
-    return token !== '';
+function formatChallenge(challenge: Array<[string, string]>): string {
+    return challenge.map((box, index) => {
+        return '[' + box.join('') + ']';
+    }).join(' ');
```

Comment:
It looks like `index` is unused, I think this can just be:

```javascript
return challenge.map(box => {
  return '[' + box.join('') + ']';
}).join(' ');
```